### PR TITLE
Powershell script to run compose in a container.

### PR DESCRIPTION
**Pull request by au-phiware**
As requested by @dnephin [here](https://github.com/docker/compose/issues/1085#issuecomment-146599493) (sorry it took so long).

This script assumes the typical environment that Windows users operate,
namely, VirtualBox running the boot2docker ISO, managed by
docker-machine. I wrote this script for my Windows using colleagues and
first placed it in the public domain as a gist:
https://gist.github.com/au-phiware/25213e72c80040f398ba
In short, that script works for me.

I have adapted that script to use the (yet to be) official image
(docker/compose:latest) and also added an extra environment variable to
provide additional options to docker run.

### CODE REVIEW

**Comment by dnephin** [comment]
Thanks for submitting this as a PR!

**Comment by thaJeztah** [comment]
Sweet! Thanks @au-phiware!

**Comment by mnowster** [comment]
LGTM. Thanks for doing this @au-phiware

**Comment by dnephin** [comment]
LGTM

It doesn't yet support all the same options as the linux/osx bash version, but that's ok for now
